docs/content/docs/dev/datastream/fault-tolerance/state.md

```diff
@@ -633,6 +633,15 @@ Updating the timestamp more often can improve cleanup speed
 but it decreases compaction performance because it uses JNI call from native code.
 The default background cleanup for RocksDB backend queries the current timestamp each time 1000 entries have been processed.
 
+Periodic compaction could speed up expired state entries cleanup, especially for state entries rarely accessed. 
```

Comment:
`Periodic compaction` could only work when TTL is enabled, I think we shall tell this clearly in the doc.

Reply:
I have added the description to the `Note` .